ctk/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -100,7 +102,9 @@ class SingleSignOnProfileVerifier(private val response: Node, private val acsUrl
      */
     private fun verifySsoAssertions() {
         val assertions = response.children("Assertion")
-        if (assertions.isEmpty()) {
+        val encryptedAssertions = response.children("EncryptedAssertion")
+
+        if (assertions.isEmpty() && encryptedAssertions.isEmpty()) {
```

Comment:
@bakejeyner will fix this to actually decrypt the encrypted assertion and verify them like the other assertion elements.